handbook/engineering/search/index.md

```diff
@@ -12,50 +12,52 @@ Sourcegraph search is:
 
+## Current focus
+
+Our current focus is documented in the [tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+
+
 ## Goals
 
-_Updated 2020-09-05_
+_Updated 2020-09-11_
 
-## [WIP] Medium term (3-6 months)
+### Scalable
 
-### Scale indexed search to 500k repositories
+- **Scale indexed search to 500k repositories**
    - **Problem:** We have customers who need Sourcegraph to scale to 500k repositories. Some parts of Sourcegraph don't work well at that scale.
    - **Outcome:** Sourcegraph can search 500k repositories in less than 300ms. This is on par with [grep.app](https://grep.app).
    - **Plan:** Incrementally add repositories to Sourcegraph.com until searches get slow or start breaking. Fix those things. Then continue adding repositories.
-   - **Owners:** TBD
+   - **Owners:** Keegan/Stefan
    - **Status:** Sourcegraph.com currently has about 100k repositories.
-
-## Short term (1-3 months)
-
-Our current focus is documented in the [tracking issue for the current milestone](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fsearch).
+   - **Estimated completion**: TODO
 
 ### Fast
 
 - **Unblock improving user perceived performance**
   - **Problem:** Our current search infrastructure relies on on-shot requests with aggressive timeouts, but it is hard to tune those timeouts for large codebases. If we make the timeouts too small, then search may not return results for needle-in-the-haystack queries (because the search timed out). If we make the timeouts too large, then search becomes slower across the board and users spend time waiting for more results when we could have returned useful results to them sooner. Detecting and handling these timeouts across multiple service boundaries is also error prone and has led to bugs.
-  - **Outcome:** 
-      - Unblock the ability to add responsive and fast-loading search results. 
+  - **Outcome:**
+      - Unblock the ability to add responsive and fast-loading search results.
       - Return results faster for large result sets, e.g. indexed repositories in very large codebases.
   - **Plan:** Streaming search
   - **Owner:** Keegan
   - **Status:** In progress
+  - **Estimated completion:** TODO
 
 ### Expressive
 
-- **Search Expressions (AND/OR/NOT)** ([term explanation](https://github.com/sourcegraph/sourcegraph/issues/13126), [term RFC](https://docs.google.com/document/d/1SHky6nodPs1w_zRXz24jB2nq5LbMCJl1u7hcLTGHDL8/edit#))
-  - **Problem:** We have customers who want to migrate from OpenGrok and other search tools, and they want to be able to do searches that are available in those tools.
-  - **Outcomes:** OpenGrok users eagerly migrate to Sourcegraph, and can run complex searches on Sourcegraph.
+- **Extend Search Query Language**
```

Comment:
Follow up from #1462: I suggested there that we just refer to our search query language rather than search expressions and and/or/not operators or the RFC. This mostly because it's succinct, and the part about and/or/not already exists